### PR TITLE
solved issue #7304

### PR DESCRIPTION
**Pull request by Dhruv110904**
solved #7304

### CODE REVIEW

File: interface/main/calendar/add_edit_event.php [hunk 1]
```diff
@@ -211,8 +211,8 @@ function DOBandEncounter($pc_eid)
 {
      global $event_date,$info_msg;
      // Save new DOB if it's there.
-     $patient_dob = trim($_POST['form_dob'] ?? '');
-     $tmph = $_POST['form_hour'] + 0;
+     $patient_dob = trim( string: $_POST['form_dob'] ?? '');
```
Comment (stephenwaite) [inline interface/main/calendar/add_edit_event.php:214]: thanks for the fix @Dhruv110904, no need for the `string` here and it's throwing a styling error with the space in front of it anyways so can just be
```
$patient_dob = trim($_POST['form_dob'] ?? '');
```

File: interface/main/calendar/add_edit_event.php [hunk 2]
```diff
@@ -211,8 +211,8 @@ function DOBandEncounter($pc_eid)
...
+     $tmph = (int)$_POST['form_hour'];
      $tmpm = $_POST['form_minute'] + 0;
```
Comment (stephenwaite) [inline interface/main/calendar/add_edit_event.php:216]: while you're at it, mind fixing this too please? :)

**Comment by Dhruv110904** [comment]
Sure, thanks for your feedback.

File: interface/main/calendar/add_edit_event.php (hunk 1 above)
Comment (Dhruv110904) [reply interface/main/calendar/add_edit_event.php:214]: fixed this issue, check PR #7628

**Pull request closed without merging**